tensorboard/uploader/proto/write_service.proto

```diff
@@ -128,3 +137,29 @@ message DeleteOwnUserRequest {
 message DeleteOwnUserResponse {
   // This is empty on purpose.
 }
+
+// Request to change the name of one experiment.
+message SetExperimentNameRequest{
```

Comment:
Consider combining these into one RPC, with a fieldmask? Like:

```proto
message UpdateExperimentMetadataRequest {
  Experiment updates = 1;
  ExperimentMetadataMask update_mask = 2;
}

message ExperimentMetadataMask {
  reserved 1 to 6;
  reserved "experiment_id", "create_time", "update_time", "num_scalars",
      "num_runs", "num_tags";
  bool name = 7;
  bool description = 8;
}
```

It seems common that the client will want to set both the name and
description at once, and doing so via two RPC roundtrips and two
database transactions (can’t parallelize because of contention) seems
wasteful.

Note that Google standard is to call the method `UpdateFoo` (not
`SetFoo`; this aligns with HTTP verbs), and to just pass the object as
both the request and the response.